src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -0,0 +1,51 @@
+package seedu.duke.parser;
+
+import seedu.duke.command.AddCommand;
+import seedu.duke.command.Command;
+import seedu.duke.module.Module;
+
+/**
+ * Parses user input.
+ */
+public class Parser {
+    /**
+     * Parses user input into module.
+     * @param fullCommand full user input command.
+     * @return the module.
+     */
+    public static Command parse(String fullCommand) {
+        String taskType;
+        String args;
+        String[] words;
+        words = fullCommand.split("\\s+",2);
+        taskType = words[0];
+        args = words[1];
```

Comment:
This variable name "args" could be more specific as well?

Reply:
Thanks for this comment! But in this case, I noticed that in the example `addressbook-level2`, the code uses two variables `commandWord` and `arguments` to split the fullCommand. Maybe in parser, variables' name need to be a little bit general cause the parser is used to process different types of command. 😄